astropy/coordinates/transformations.py

```diff
@@ -347,23 +347,34 @@ def get_transform(self, fromsys, tosys):
             self._composite_cache[fttuple] = comptrans
         return self._composite_cache[fttuple]
 
-    def lookup_name(self, name):
+    def lookup_name(self, name, raise_exception=False):
```

Comment:
I would prefer an argument name that is more descriptive, maybe something like `fail_if_none`?